<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete</class>
 <widget class="QWidget" name="Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tab_del">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>删除学生信息</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,10,10,15,15,15">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QCheckBox" name="check_id">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>  ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_1">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QCheckBox" name="check_name">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>姓名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_type">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>专业</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                  </font>
                 </property>
                 <property name="currentIndex">
                  <number>-1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>本科生</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>研究生</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>博士生</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="check_pro">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>专业</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="del_p1_search">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>搜 索</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="del_p1_del">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>删 除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="del_p1_cel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>撤 销</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_del1"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>删除个人成绩</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="4,10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="65,10,10,10,0,0,65,10,10,10,10,10,5">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>ID</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_id2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QCheckBox" name="check_sub">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>学科</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_sub">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>    类别</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="currentText">
                <string/>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <item>
                <property name="text">
                 <string>本科生</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>研究生</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>博士生</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="del_p2_search">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>搜 索</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="del_p2_del_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>删 除</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="del_p2_cel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>撤 销</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>189</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_del2"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="10,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="del_ret">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
